README.md

```diff
@@ -29,7 +29,7 @@ performance = predictor.evaluate(test_data)
 See the [AutoGluon Website](http://autogluon.mxnet.io/index.html) for [documentation](https://autogluon.mxnet.io/api/index.html) and instructions on:
 - [Installing AutoGluon](http://autogluon.mxnet.io/index.html#installation)
 - [Learning with tabular data](http://autogluon.mxnet.io/tutorials/tabular_prediction/tabular-quickstart.html)
-  - [Tips to maximize accuracy](https://autogluon.mxnet.io/tutorials/tabular_prediction/tabular-indepth.html#maximizing-predictive-performance)
+  - [Tips to maximize accuracy](http://autogluon.mxnet.io/tutorials/tabular_prediction/tabular-quickstart.html#maximizing-predictive-performance)
```

Comment:
I don't see you moving this section in this PR, did you intend to?

Reply:
yes it has been moved to quickstart tutorial